src/components/creator/steps/details.tsx

```diff
@@ -0,0 +1,139 @@
+import { ItemKind, ItemMeta, isItemKind, metaForKind } from '../meta';
+import {
+  ConditionProp,
+  Field,
+  componentTypes,
+  dataTypes,
+  validatorTypes,
+} from '@data-driven-forms/react-form-renderer';
+import React from 'react';
+import { ChromeAPI } from '@redhat-cloud-services/types';
+import {
+  NAME_BUNDLES,
+  NAME_DESCRIPTION,
+  NAME_DURATION,
+  NAME_KIND,
+  NAME_TITLE,
+  NAME_URL,
+  REQUIRED,
+} from './common';
+import { panelOverviewStepName } from './panel-overview';
+
+export type Bundles = ReturnType<ChromeAPI['getAvailableBundles']>;
+
+function kindMetaCondition(test: (meta: ItemMeta) => boolean): ConditionProp {
+  return {
+    when: NAME_KIND,
+    is: (kind: string | undefined) => {
+      return (
+        typeof kind === 'string' && isItemKind(kind) && test(metaForKind(kind))
+      );
+    },
+  };
+}
+
+const DETAILS_STEP_PREFIX = 'step-details-';
+
+export function isDetailsStep(name: string): boolean {
+  return name.startsWith(DETAILS_STEP_PREFIX);
+}
+
+export function detailsStepName(kind: ItemKind): string {
+  return `${DETAILS_STEP_PREFIX}${kind}`;
+}
+
+export function makeDetailsStep({
+  kind,
+  bundles,
+  downloadStep,
+}: {
+  kind: ItemKind;
+  bundles: Bundles;
+  downloadStep: string;
+}) {
+  const meta = metaForKind(kind);
+
+  const fields: Field[] = [];
+
+  fields.push(
+    {
+      component: componentTypes.PLAIN_TEXT,
+      name: 'internal-text-details-description',
+      label:
+        'Share the details required to populate a card in the correct places.',
+    },
+    {
+      component: componentTypes.SELECT,
+      name: NAME_BUNDLES,
+      label: 'Associated bundle(s)',
```

Comment:
This component is missing the tooltip icon:
![screenshot-www_figma_com-2024_07_17-10_17_12](https://github.com/user-attachments/assets/3f329247-94ff-4d51-aa86-f9cd22bd2ca1)

You should take to to UX about the tooltip/popover content.

Reply:
Talked to UX, it is now removed.